### DISCUSSION TITLE
Requested changes date is listed as drafting date

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#146598] [Mobile] [answered]
GitHub Mobile beta for Android 

This notification: https://github.com/pjcj/Devel--Cover/pull/339#pullrequestreview-2190386528

Shows up as recent: 
![Screenshot_20241208-062122.png](https://github.com/user-attachments/assets/4770ee8f-2acf-4bb1-ac55-750e5ec63cdf)

But when opened, shows it was nonsensically written 5 months ago with an apology about using GitHub: 
![Screenshot_20241208-061832.png](https://github.com/user-attachments/assets/86b5ab3b-c140-4013-ab06-59799cb9a66b)

Whereas it shows it's recent in the web view:
![Screenshot_20241208-061853.png](https://github.com/user-attachments/assets/76945c7d-59d0-456e-8800-e1ae37aa3458)

**Comment by taki-on** [comment, marked as answer]
Thank you for reporting the bug @jsoref . The date seems odd indeed. I also confirmed that it happens on iOS, too. I will file a bug.
